### PR TITLE
Adapted to kube-rs 0.60 and operator-rs 0.2.2

### PR DESCRIPTION
**Pull request by maltesander**
## Review Checklist
- [x] Code contains useful comments
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ All notable changes to this project will be documented in this file.
 - Added `ProductVersion` to status ([#81]).
 - Added `Condition` to status ([#81]).
 
+### Changed
+
+- `kube-rs`: `0.58` → `0.60` ([#83]).
+- `k8s-openapi` `0.12` → `0.13` and features: `v1_21` → `v1_22` ([#83]).
+- `operator-rs` `0.2.1` → `0.2.2` ([#83]).
```
Comment (lfrancke) [inline CHANGELOG.md:16]: It's a bit too late now I guess but I only now realize: The upgrade from 0.58 to 0.59 was a BREAKING change, that should have meant we do a version bump from 0.2 to 0.3 in operator-rs. Next time.

File: rust/operator-binary/src/stackable-nifi-operator.rs [hunk 2]
```diff
@@ -49,17 +47,5 @@ async fn main() -> Result<(), error::Error> {
 
     let client = client::create_client(Some("nifi.stackable.tech".to_string())).await?;
 
-    if let Err(error) = stackable_operator::crd::wait_until_crds_present(
```
Comment (lfrancke) [inline rust/operator-binary/src/stackable-nifi-operator.rs:52]: Why did you move this to the `create_controller` function?
I'm not really opposed, happy to merge as is, just wondering.

**Review by lfrancke — APPROVED** [review]
Approved with comments that don't need action

File: rust/operator-binary/src/stackable-nifi-operator.rs (hunk 2 above)
Comment (maltesander) [reply rust/operator-binary/src/stackable-nifi-operator.rs:52]: I would required the kube-rs and k8s-openapi dependency in the binary as well.
